package.json

```diff
@@ -81,12 +81,12 @@
   },
   "dependencies": {
     "classnames": "^2.2.0",
-    "cozy-ui": "^3.0.0-beta24",
+    "cozy-ui": "beta32",
     "date-fns": "^1.27.2",
     "node-polyglot": "^2.2.0",
     "piwik-react-router": "^0.8.2",
-    "preact": "^7.1.0",
-    "preact-compat": "^3.9.0",
+    "preact": "^8.1.0",
```

Comment:
Ok, good to know so. But here I think that will be fine, thanks @goldoraf  and @GoOz 👍